scanner/scanner.go

```diff
@@ -390,6 +392,17 @@ func isIgnoredField(f *types.Var, tags []string) bool {
 	return !f.Exported() || (len(tags) > 0 && tags[0] == "-")
 }
 
+func getProtoID(f *types.Var, tags []string) int {
+	if len(tags) == 0 {
+		return 0
+	}
+	i, err := strconv.Atoi(tags[0])
```

Comment:
I think it may be better to use `proteus:",101"` notation - it will be easier to extend later.

Reply:
Thank you for your suggestion, as you wish.